src/components/02_atoms/Widgets/BooleanCheckbox.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CheckBox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import WidgetPropTypes from '../../05_pages/NodeForm/WidgetPropTypes';
+
+const BooleanCheckbox = props => {
+  const { onChange, label, id, value, ...checkboxProps } = props;
+
+  return (
+    <FormControlLabel
+      id={`${props.fieldName}-label`}
+      control={
+        <CheckBox
+          id={`${props.fieldName}-cb`}
+          onChange={event => onChange(event.target.value)}
```

Comment:
Note: for setting the right checked value, you need to access ```event.target.checked```:

```
event.target.value
""
event.target.checked
true
```

Reply:
Interesting, ```value``` returns "true" or "", while ```checked``` returns ```true``` or ```false```.

Changed!